src/NvGet/Helpers/SolutionHelper.cs

```diff
@@ -173,7 +176,7 @@ private static async Task<PackageReference[]> GetFileReferences(CancellationToke
 
 			if(target.HasAnyFlag(FileType.Csproj, FileType.DirectoryProps, FileType.DirectoryTargets, FileType.CentralPackageManagement))
 			{
-				references = document.GetPackageReferences();
+				references = document.GetPackageReferences().Concat(document.GetProjectProperties(projectProperties)).ToArray();
```

Comment:
It's not impossible to define the property in any `.props`/`.targets`. I think we currently don't do that, so it's not an important case (assuming the tool is intended for internal usage only)

Reply:
Yeh didn't want to change anything else in terms of supported files etc